.github/ISSUE_TEMPLATE/if-plugin-registry-form-template.md

```diff
@@ -0,0 +1,32 @@
+---
+name: IF Plugin Registry Form template
+about: Use this template to submit an IF plugin for registry
+title: "[PLUGIN REGISTRY]"
+labels: ''
+assignees: ''
+
+---
+
+## Plugin Name
+<!--- How do you call your plugin? -->
+
+## Plugin GitHub URL
+<!--- URL -->
+
+## Plugin Registry Basic Requirements
+- [ ] My plugin executes a demo manifest yml with no errors or crashes.
+- [ ] My plugin has README file.
+- [ ] My plugin code includes a sample manifest .yaml file.
+- [ ] My plugin's code contain unit tests that cover >=80% of the plugins code.
+- [ ] All unit tests that test my plugin are passing.
+- [ ] All other unit tests in the if-models repo are passing after my code is integrated into it.
```

Comment:
```suggestion
```

`if-models` is now `if-plugins`, and anyway the point of the plugin registry is that plugins won't make it into that repository - they will stay external. So I think this should just get deleted.